### DISCUSSION TITLE
my piano file is missing monika says I can play the piano now but im missing the zz_pianokeys.rpyc file can someone give it to me please :((

### DISCUSSION THREAD

**Discussion by lolhope0** [Monika-After-Story/MonikaModDev#9959] [general - piano notes]
<img width="1336" alt="Screenshot 2023-04-12 at 5 27 15 PM" src="https://user-images.githubusercontent.com/130611876/231599882-d8c5b7e4-188c-4824-a3af-21b0bf04f28b.png"<img width="1332" alt="Screenshot 2023-04-12 at 5 27 26 PM" src="https://user-images.githubusercontent.com/130611876/231599904-86dbe1da-ceb7-42d6-812f-ebfb580e9095.png">

**Comment by Wingdinggaster656** [comment]
`zz_pianokeys.rpyc` should have been naturally attached into the game pack. You probably had a bad installation, consider reinstalling.

**Reply by lolhope0** [reply]
thank you this helped :)
